src/axom/quest/tests/quest_sampling_shaper.cpp

```diff
@@ -166,14 +166,28 @@ class SamplingShaperTest : public ::testing::Test
     EXPECT_NE(nullptr, m_shaper)
       << "Shaper needs to be initialized via initializeShaping()";
 
-    const auto shapeDim = m_shapeSet->getDimensions();
+    // Define lambda to override default dimensions, when necessary
+    auto getShapeDim = [defaultDim =
```

Comment:
This map sets/overrides the dimension for each shape, rather than for the entire problem. It was the only necessary modification required to support using `c2c` and `stl` meshes in the same deck.